test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -1276,6 +1276,11 @@ protected com.datastax.driver.core.ResultSet executeNetWithPaging(String query,
         return sessionNet().execute(new SimpleStatement(formatQuery(query)).setFetchSize(pageSize));
     }
 
+    protected com.datastax.driver.core.ResultSet executeNetWithoutPaging(String query)
+    {
+        return executeNetWithPaging(query, Integer.MAX_VALUE);
+    }
+
```

Comment:
Why did you introduce that method?

Reply:
`executeNet` uses the default page size, whereas `executeNetWithPaging` allows to specify an ad-hoc page size for the query. The new method is a shortcut to run the query with paging disabled, which is required by post-ordering.

Currently the only caller is `ColumnMaskTest`, but I think it can be useful for other tests to have that utility method on `CQLTester`.